### PR TITLE
Fix Cloudburst Nukkit download links

### PR DESCRIPTION
**Pull request by MrRazamataz**
### All Submissions:

* [x ] Have you followed the guidelines in our Contributing document?
* [x ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [ ] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:
Just a quick fix, no new files.

1. [x ] Have you added an explanation of what your changes do and why you'd like us to include them? 
its below
2. [x ] Have you tested your Egg changes?

The download URL for the nukkit CI seems to have changed, so I updated that and a few other links that were dead.
With the new links it seems older versions cant be downloaded, so I left them as-is
Also, there is no eula promt for nukkit, so thats yeeted.

### CODE REVIEW

File: game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json [hunk 1]
```diff
@@ -4,25 +4,29 @@
...
+        "ghcr.io\/pterodactyl\/yolks:java_16",
+        "ghcr.io\/pterodactyl\/yolks:java_8",
+        "ghcr.io\/pterodactyl\/yolks:java_17"
     ],
     "file_denylist": [],
     "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}}",
     "config": {
         "files": "{\r\n    \"server.properties\": {\r\n        \"parser\": \"properties\",\r\n        \"find\": {\r\n            \"server-ip\": \"0.0.0.0\",\r\n            \"enable-query\": \"true\",\r\n            \"server-port\": \"{{server.build.default.port}}\",\r\n            \"query.port\": \"{{server.build.default.port}}\"\r\n        }\r\n    }\r\n}",
-        "startup": "{\r\n    \"done\": \")! For help, type \",\r\n    \"userInteraction\": [\r\n        \"Go to eula.txt for more info.\"\r\n    ]\r\n}",
+        "startup": "{\r\n    \"done\": \")! For help, type \"\r\n}",
```
Comment (parkervcp) [inline game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json:23]: Why remote the user interaction here?

File: game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json [hunk 2]
```diff
@@ -4,25 +4,29 @@
...
...ript": "#!\/bin\/ash\r\n# Nukkit Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napk add --no-cache curl openssl\r\n\r\ncd \/mnt\/server\r\n\r\n## Only download if a path is provided, otherwise continue.\r\nif [ ! -z \"${DL_PATH}\" ]; then\r\n    MODIFIED_DOWNLOAD=`eval echo $(echo ${DL_PATH} | sed -e 's\/{{\/${\/g' -e 's\/}}\/}\/g')`\r\n    wget ${MODIFIED_DOWNLOAD} -O ${SERVER_JARFILE}\r\nelif [ -z \"${NUKKIT_VERSION}\" ] || [ \"${NUKKIT_VERSION}\" == \"latest\" ]; then\r\n    wget https:\/\/repo.opencollab.dev\/artifactory\/maven-snapshots\/cn\/nukkit\/nukkit\/1.0-SNAPSHOT\/nukkit-1.0-SNAPSHOT.jar -O ${SERVER_JARFILE}\r\nelse\r\n    wget https:\/\/ci.opencollab.dev\/job\/NukkitX\/job\/Nukkit\/job\/master\/${NUKKIT_VERSION}\/artifact\/target\/nukkit-1.0-SNAPSHOT.jar -O ${SERVER_JARFILE}\r\nfi\r\n\r\nif [ ! -f nukkit.yml ]; then\r\n    wget https:\/\/raw.githubusercontent.com\/parkervcp\/eggs\/master\/game_eggs\/minecraft\/bedrock\/nukkit\/nukkit.yml\r\nfi\r\n\r\nif [ ! -f server.properties ]; then\r\n    wget https:\/\/raw.githubusercontent.com\/parkervcp\/eggs\/master\/game_eggs\/minecraft\/bedrock\/nukkit\/server.properties\r\nfi\r\necho -e \"Install completed!\"",
```
Comment (parkervcp) [inline game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json:29]: I disagree with editing the download to the repo page when getting the latest from the jenkins page is perfectly valid.

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json (hunk 1 above)
Comment (Software-Noob) [reply game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json:23]: It's deprecated and not used anymore. Egg features have taken this role

File: game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json (hunk 2 above)
Comment (MrRazamataz) [reply game_eggs/minecraft/bedrock/nukkit/egg-nukkit.json:29]: It appears the CI for it is now correctly displaying the build artifacts again, I'll take a look at amending these issues later today

**Comment by MrRazamataz** [comment]
That has reverted to the original URL, as it's working again.
The other changes have fixed.

**Review by parkervcp — APPROVED** [review]

**Pull request merged**